addons/mrp/models/product.py

```diff
@@ -22,11 +22,26 @@ class ProductTemplate(models.Model):
     produce_delay = fields.Float(
         'Manufacturing Lead Time', default=0.0,
         help="Average lead time in days to manufacture this product. In the case of multi-level BOM, the manufacturing lead times of the components will be added.")
+    is_kits = fields.Boolean(compute='_compute_is_kits', compute_sudo=False)
```

Comment:
Any reason for the compute_sudo ? In case you open the product form it may be problematic if you don't have right access on mrp

Reply:
yep, in enterprise https://github.com/odoo/enterprise/pull/20440/commits/0bda231d6064b3e28e55c4372a600e6a3dc2771c

Reply:
compute_sudo = True if field is store=True, False if store=False. In that case, it's useless.

I think it should be available as stat button but I'm not sure user should be allow to click on it.

Reply:
You are right, confused myself with related compute field

Reply:
I think I followed this https://github.com/odoo/odoo/commit/5d75d07bad44bd542bb82025e465c798326f4b92

Reply:
I don't have any access error with a non mrp user but I don't get the quantity buttons neither. I don't know if I should have it or not